slack-notifier/README.md

```diff
@@ -15,7 +15,24 @@ Follow these steps to configure the webhook in Slack:
 
 4. Copy the webhook URL from the setup instructions and use it in the AWS Setup section that follows.
 
-### AWS Setup
+
+### AWS setup using CloudFormation 
+
+#### CloudFormation
+Choose **Launch Stack** to launch the AWS Health SMS Notifier template in the US East (N. Virginia) Region in your account:
+ 
+
+<a href="https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=SlackNotifier&templateURL=https://s3.amazonaws.com/aws-health-tools/Cloudformation-templates/slack-notifier.json" title="Launch Stack"><img src="launch_stack.png" alt="Launch Stack" /></a>
+ 
+The CloudFormation template requires the following parameters:
+ 
+*SlackChannel* - A Slack channel name where you would like the notifications to get pushed.
+ 
+*HookURL* - Incoming web hook url from Slack setup.
+
+
+
+### AWS Manual Setup
```

Comment:
I'm not too keen on giving customers the option of manual steps. It's too error-prone and an additional thing to maintain (in case we need to update the CFN template). I would delete this section in favor of the CFN deployment.